packages/unmock-core/src/service/interfaces.ts

```diff
@@ -55,12 +62,19 @@ export interface IServiceInput {
   name: string;
 }
 
+//           (e.g. "application/json")   schema
+export type mediaTypeToSchema = Record<string, Schema>;
+//                              status   (e.g. "application/json")
```

Comment:
`mediaTypeToSchema` here is not `application/json` as the comment says but a map with possible key of `application/json`, I assume?

Reply:
Yup, that's what I meant by the comment (`string` being a media type, e.g. `"application/json"`). Sorry for the lack of clarity there :)

Reply:
Will update those comments to make it clearer.